versions.txt

```diff
@@ -23,11 +23,11 @@ autoinstrumentation-nodejs=0.51.0
 
 # Represents the current release of Python instrumentation.
 # Should match value in autoinstrumentation/python/requirements.txt
-autoinstrumentation-python=0.44b0
+autoinstrumentation-python=0.45b0
 
 # Represents the current release of DotNet instrumentation.
 # Should match autoinstrumentation/dotnet/version.txt
-autoinstrumentation-dotnet=1.2.0
+autoinstrumentation-dotnet=1.6.0
```

Comment:
No don't do this yet, this is the version with breaking changes to http semantic conventions, and we still havent decided when to take that hit by default